s3/s3_operations.cpp

```diff
@@ -120,20 +123,103 @@ namespace irods_s3 {
             int fd_counter;
     };
 
-    int debug_log_level = LOG_DEBUG;
+    int debug_log_level = LOG_NOTICE;
```

Comment:
is this going to be too verbose for production deployments?

Reply:
This is correct.  I missed this setting.  Updated.